api/src/beacons/dto/find-beacons.dto.ts

```diff
@@ -16,4 +17,9 @@ export class FindBeaconsDto {
   @IsNotEmpty()
   @IsNumber()
   maxDistance: number;
+
+  @IsOptional()
+  @Transform(({ value }) => Tags[value])
+  @IsEnum(Tags)
+  tag: Tags;
```

Comment:
Have you tested this? I think it should be the following.
`@IsOptional()
@Type(() => Status)
@IsArray()
@ValidateNested({ each: true })
@IsEnum(Tag, { each: true })
tags: Tag[];`

Reply:
I had this set up to only allow for searching by one beacon at a time, if not I'm gonna need to change the find function as well, but it makes sense to be able to search for multiple so I'll do that.

Reply:
Where should I be importing Status from? I can't find a reference to it in either of the modules you otherwise referenced in these files.